src/templates/api.tsx

```diff
@@ -9,6 +9,7 @@ import {
 } from '../components';
 import { apiMdxComponents } from '../mdxComponents';
 import ApiNavigation from '../navigations/api';
+import JsonLink from '../components/ApiComponents/Components/JsonLink';
```

Comment:
You shouldn't import this directly, it should be `ApiComponents.JsonLink` that's why I mentioned updating `src/components/ApiComponents/Components/index.ts` and add the export to your component there 👀

Reply:
Apologies for missing that bit of the feedback!